tests/tasks/exec/start_stop_exec.cue

```diff
@@ -135,14 +139,13 @@ dagger.#Plan & {
 					contents: sharedCache
 				}
 				args: [
-					"sh", "-e", "-c",
+					"sh", "-x", "-e", "-c",
```

Comment:
`-x` stands for debug purpose right? Is it still useful there?

Reply:
Yeah it results in the actual commands being printed as they are executed. I'd like to keep it in place. If the test passes, no harm done; if the test fails then they could be helpful for debugging.